styles/globals.scss

```diff
@@ -1,3 +1,5 @@
+@import "./tools/functions";
```

Comment:
We'll also need to import the `_colors.scss` file here, otherwise the function won't be able to find the `$colors` map.

All the variables should be imported first, then any functions that depend on those variables, and then the rest of our SCSS will follow.

Reply:
Taken care of and that makes sense why you have to do that .